openedx/core/djangoapps/catalog/models.py

```diff
@@ -29,6 +29,15 @@ class CatalogIntegration(ConfigurationModel):
         )
     )
 
+    long_term_cache_ttl = models.PositiveIntegerField(
+        verbose_name=_('Long Term Cache Time To Live'),
+        default=0,
```

Comment:
Do we want a better default here?  Like 24 hours.

Reply:
I am a bit torn here.  I think 0 means it will not add caching to those that don't want it, but you have to explicitly use the long_term TTL so it would be helpful.

Reply:
but the default to use is False.  So setting 0 here is probably not what we want.